### PR TITLE
Add format to BarcodeValue

### PR DESCRIPTION
**Pull request by jeremy-dowdall**
fix for #61

### CODE REVIEW

File: lib/src/vision/barcode_scanner.dart [hunk 1]
```diff
@@ -185,6 +185,11 @@ class BarcodeValue {
   ///
   final BarcodeType type;
 
+  /// The format (symbology) of the barcode value.
+  ///
+  /// For example, [BarcodeFormat.upca], [BarcodeFormat.code128], [BarcodeFormat.dataMatrix]
+  final int format;
```
Comment (fbernaly) [inline lib/src/vision/barcode_scanner.dart:191]: Change this to an enum

**Comment by fbernaly** [comment]
@jeremy-dowdall : thanks for contributing

File: lib/src/vision/barcode_scanner.dart (hunk 1 above)
Comment (jeremy-dowdall) [reply lib/src/vision/barcode_scanner.dart]: 👍 it does work much nicer as an enum. updated in ad1e5c4

**Comment by fbernaly** [comment]
@jeremy-dowdall: the static analysis is failing. Run these commands to fix the code and commit the changes:
```
flutter format --set-exit-if-changed .
flutter analyze .
```

**Review by fbernaly — APPROVED** [review]

**Pull request merged**